content/setup/local-with-vagrant/_index.md

```diff
@@ -0,0 +1,198 @@
+
+## Cloning Tinkerbell
+
+1. Clone `tink` and move inside `deploy/vagrant` directory. This folder contains
+   vagrant configuration (Vagrantfile) and the scripts needed to provide workers
+   and provisioners.
+
+```
+$ git clone https://github.com/tinkerbell/tink.git
+$ cd tink/deploy/vagrant
+```
+
+## Start the provisioner
+
+```
+$ vagrant up provisioner
+Bringing machine 'provisioner' up with 'virtualbox' provider...
+
+    ......
+
+    INFO: tinkerbell stack setup completed successfully on ubuntu server
+    NEXT: 1. Enter ./deploy and run: source ../envrc; docker-compose up
+           2. Try executing your fist workflow.
+              Follow the steps described in https://tinkerbell.org/examples/hello-world/ to say 'Hello World!' with a workflow.
+```
+
+When the provisioner is ready, a summary is printed to the console and you can now ssh in.
+
+### Connecting to the Provisioner
+
+All the following commands have to be executed in the artifacts' directory created above unless stated otherwise.
+
+Connect to the provisioner via `vagrant ssh`:
+
+```
+$ vagrant ssh provisioner
+vagrant@provisioner:~$
+```
+
+At this point, you are in an Ubuntu box that has a couple of utils installed.
+
+Let's now start up the Tinkerbell stack with `docker-compose`:
+
+```
+$ sudo su -l
+$ cd /vagrant && source envrc && cd deploy
+$ docker-compose up -d
+```
+
+> Note: this is now managed like a standard Docker Compose project. Just make sure
+> to have sourced the `envrc` before issuing `docker-compose commands.`
+
+Those commands give you a fully working provisioner that is ready to receive
+templates and workflow targeting workers. You can check that it is running:
+
+```
+$ docker-compose ps
+        Name                      Command                  State                             Ports
+-------------------------------------------------------------------------------------------------------------------------
+deploy_boots_1         /boots -dhcp-addr 0.0.0.0: ...   Up
+deploy_cacher_1        /cacher                          Up             0.0.0.0:42111->42111/tcp, 0.0.0.0:42112->42112/tcp
+deploy_db_1            docker-entrypoint.sh postgres    Up (healthy)   0.0.0.0:5432->5432/tcp
+deploy_hegel_1         cmd/hegel                        Up
+deploy_nginx_1         /docker-entrypoint.sh ngin ...   Up             192.168.1.2:80->80/tcp
+deploy_registry_1      /entrypoint.sh /etc/docker ...   Up (healthy)
+deploy_tink-cli_1      /bin/sh -c sleep infinity        Up
+deploy_tink-server_1   tink-server                      Up (healthy)   0.0.0.0:42113->42113/tcp, 0.0.0.0:42114->42114/tcp
+```
+
+### Register the worker's hardware
+
+The provisioner is running we can follow the [example called "Hello
+world"](/examples/hello-world), I will contextualize it for our vagrant setup
+here.
+
+SSH in the provisioner again and we have to define our hello world template and load it to Tinkerbell:
+
+```
+$ cat > hello-world.yml
```

Comment:
If you're using `cat >` then you really need to use the EOF syntax.

https://stackoverflow.com/questions/2500436/how-does-cat-eof-work-in-bash

Regular users won't know to hit control + d.